app/src/main/java/com/xmartlabs/template/analytics/AnalyticsManager.java

```diff
@@ -0,0 +1,33 @@
+package com.xmartlabs.template.analytics;
+
+import android.support.annotation.NonNull;
+
+import com.annimon.stream.Stream;
+import com.xmartlabs.base.core.log.analytics.AnalyticTrackable;
+import com.xmartlabs.base.core.log.analytics.AnalyticTracker;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+public final class AnalyticsManager {
```

Comment:
thinking with @diegomedina248 we think that it could be better it isin the core. What do you think?